cmd/query/app/http_handler.go

```diff
@@ -266,7 +266,10 @@ func (aH *APIHandler) tracesByIDs(ctx context.Context, traceIDs []model.TraceID)
 	var traceErrors []structuredError
 	retMe := make([]*model.Trace, 0, len(traceIDs))
 	for _, traceID := range traceIDs {
-		if trc, err := aH.queryService.GetTrace(ctx, traceID); err != nil {
+		query := spanstore.GetTraceParameters{
```

Comment:
somewhere here the timestamps must be handled, so leave a TODO. This could be another PR to precede this one, where we introduce timestamps params in /api/trace/{traceID} API.

Reply:
how is this comment resolved?

Reply:
Added more TODOs